Modules/Files/test/Files.API.Tests.Performance.EndToEnd/prettierrc.json

```diff
@@ -0,0 +1,15 @@
+{
+    "endOfLine": "lf",
+    "printWidth": 120,
+    "tabWidth": 4,
+    "trailingComma": "none",
+    "semi": false,
```

Comment:
Set it to true (I forgot that they don't use semicolons for most of the enmeshed consumption projects)